evap/student/forms.py

```diff
@@ -12,8 +12,45 @@ def __init__(self, label):
         super().__init__(label=label, required=False)
 
 
+class TextAnswerField(forms.CharField):
+
+    def __init__(self, related_question_id="", *args, **kwargs):
+        # pylint: disable=keyword-arg-before-vararg
```

Comment:
could we change it to
```
    def __init__(self, *args, related_question_id="", **kwargs):
```
? Same below.

Reply:
i copied it from the django docs but sure, they should better be swapped